.github/workflows/unit-test.yml

```diff
@@ -22,32 +26,44 @@ jobs:
       - name: List Project Configuration
         run: |
           xcodebuild -list
-      - name: iOS Tests
+      - name: ${{ matrix.platform }} Tests
         run: |
-          xcodebuild test \
-            -scheme Amplitude-Swift-Package \
-            -sdk iphonesimulator \
-            -destination 'platform=iOS Simulator,name=iPhone 15'
-      - name: macOS Tests
-        run: |
-          xcodebuild test \
-            -scheme Amplitude-Swift-Package \
-            -sdk macosx \
-            -destination 'platform=macosx'
-      - name: tvOS Tests
-        run: |
-          xcodebuild \
-            -scheme Amplitude-Swift-Package \
-            -sdk appletvsimulator \
-            -destination 'platform=tvOS Simulator,name=Apple TV' \
-            test
-      - name: watchOS Tests
+          case "${{ matrix.platform }}" in
+            iOS)
+              xcodebuild test \
+                -scheme Amplitude-Swift-Package \
+                -sdk iphonesimulator \
+                -destination 'platform=iOS Simulator,name=iPhone 15'
```

Comment:
can we add `-version` for each platform based on https://github.com/amplitude/Amplitude-Swift/blob/df08b88cca1035d36b54efa01ce0d2b4ad43419d/Package.swift#L8-L13

Reply:
@Mercy811 - I think the `latest` version is sufficient for most cases. We can of course add more tests to cover older OS versions later.

Reply:
is latest consistent with the current behavior?

Reply:
yes, I believe `latest` is the current behavior since no OS version is specified in the test commands.